### ISSUE TITLE
Generate more project files with `golo new`

### ISSUE THREAD

**Issue by yloiseau** [eclipse/golo-lang#473]
The `golo new` command could generate more files to help initialize a new golo project matching the common practices. For instances:
- [ ] `CHANGELOG.md` file stub
- [ ] `LICENCE` file (and add the values in the gradle/maven build files): I'd suggest to use EPL by default
- [ ] `doc/main.adoc` (or markdown), updating the gradle file with the asciidoctor plugin e.g.
- [ ] `src/test/golo/main.golo` stub (when a default test framework will be ready)

and probably more. I'm not sure it's relevant. Open to discussion.

**Comment by utybo** [comment]
I strongly disagree with having the EPL as the default. I think there should not be any default, but that you should rather ask the user for a licence, and/or redirect to a site like https://choosealicense.com/ or just not create any LICENCE file at all and simply reming the user through in the "golo new" that they should choose a licence for their project and create a LICENCE file.
